src/iks/create_ingress.py

```diff
@@ -0,0 +1,109 @@
+import requests
+import json
+import os
+import subprocess
+from src.common.functions import Color as Color
+
+class IngressCreator:
+    def __init__(self, clusterNameOrID, resourceGroupID, namespace, secretName, serviceName, servicePort, accessToken, refreshToken ):
+        self.clusterNameOrID=clusterNameOrID
+        self.resourceGroupID=resourceGroupID
+        self.namespace=namespace
+        self.secretName=secretName
+        self.serviceName=serviceName
+        self.servicePort=servicePort
+        self.accessToken= accessToken
+        self.refreshToken=refreshToken
+        print("values: ",clusterNameOrID,resourceGroupID,namespace,secretName,serviceName,servicePort)
+    def getIngressSubdomain(self):
```

Comment:
I actually have a function already to get the ingress subdomain. It's get_iks_info in function.py. I save the ingress subdomain in our app_url variable there. My bad for the ambiguous wording, but since my function is actually a more generalized request, I thought get_iks_info could be used to get other information from the cluster if needed.

Reply:
I believe that using GET /v1/clusters/{ID} is a more reliable way to get the ingress subdomain. I can replace your function with mine.